docs/faqs/faqs.md

```diff
@@ -0,0 +1,212 @@
+---
+id: faqs
+title: Frequently Asked Questions
+sidebar_label: FAQs
+slug: faqs/faqs
+---
+
+
+# FAQs
+
+These are some of the questions we are often asked about using Amplication
+
+
+<details>
+  <summary>What is Amplication?</summary>
+
+  Amplication is an open-source development tool. It helps you develop quality Node.js applications without spending time on repetitive coding tasks. Amplication auto-generates fully functional apps based on TypeScript and Node.js.
```

Comment:
Amplication is an open-source platform that helps developers build backend services without spending time on repetitive coding tasks and boilerplate code. Amplication auto-generates fully functional, production-ready backend based on TypeScript and Node.js.